File: my-folder/0026-remove-duplicates-from-sorted-array/solution.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if (nums is null) return 0;

        int length = nums.Length;
        if (length == 1) return 1;

        int index = 0;

        for (int i = 0; i < length - 1; i++) {
            if (nums[i] != nums[i + 1]) {
                nums[index++] = nums[i];
            }
        }

        nums[index++] = nums[^1];

        return index;
    }
}

File: my-folder/0026-remove-duplicates-from-sorted-array/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if (!nums?.length) return 0;
    
    const n = nums.length;
    if (n === 1) return 1;

    let index = 0;

    for (let i = 0; i < n - 1; i++) {
        if (nums[i] !== nums[i + 1]) {
            nums[index++] = nums[i];
        }
    }

    nums[index++] = nums[n - 1];

    return index;
};
